ko/admin/admin_utils.rst

```diff
@@ -3664,6 +3665,67 @@ flashback
 
         cubrid flashback --oldest demodb dba.tbl
 
+.. _memmon:
+
+memmon
+---------
+
+ **cubrid memmon** 유틸리티는 현재 서버 프로세스에 할당된 힙 메모리 사용량을 출력한다. 시스템 파라미터 **enable_memory_monitoring** 가 yes로 설정된 경우, 서버 메모리 모니터링 모듈은 힙 메모리 총 사용량과 메모리 할당이 발생한 큐브리드 소스 코드 및 라인 정보를 기준으로 세부적인 메모리 할당 정보를 추적 관리한다. 이를 통해 유틸리티를 실행하는 시점의 서버 힙 메모리 사용 현황을 확인할 수 있다. ::
+
+    cubrid memmon [option] database_name
+
+*   **cubrid**: CUBRID 서비스 및 데이터베이스 관리를 위한 통합 유틸리티
+
+*   **memmon**: 서버의 힙 메모리 사용 현황을 출력하는 유틸리티
+
+*   *database_name*: 힙 메모리 사용 현황을 확인할 데이터베이스 이름
+
+다음 예제에서는 사용자가 "cubrid memmon demodb"를 수행했을 때 출력되는 결과를 보여준다.
+
+::
+
+    ====================cubrid memmon====================
+    Server Name: demodb
+    Total Memory Usage: 935240 KB (for meta info: 470 KB)
+    -----------------------------------------------------
+            File Name                                                                                           |     Memory Usage(Ratio)
+            storage/page_buffer.c:4977                                                                          |           525315 KB( 56%)
+            transaction/log_page_buffer.c:584                                                                   |           262147 KB( 28%)
+            ...
+
+**cubrid memmon** 유틸리티를 실행하면 데이터베이스 이름, 힙 메모리 총 사용량, 할당된 힙 메모리를 추적 관리하기 위해 서버 내부적으로 유지되는 메타 정보의 메모리 사용량이 출력된다. 세부 항목으로는 힙 메모리 할당이 발생한 모든 파일과 라인 별[파일:라인] 메모리 사용량과 점유율이 사용량 기준으로 정렬되어 출력된다. 만약 --disable-force 옵션을 이용하여 메모리 모니터링 기능을 강제로 중지시킨 경우, 아무것도 출력되지 않는다.
```

Comment:
```suggestion
**cubrid memmon** 유틸리티를 실행하면 데이터베이스 이름, 힙 메모리 총 사용량, 할당된 힙 메모리를 추적 관리하기 위해 서버 내부적으로 유지되는 메타 정보의 메모리 사용량이 출력된다. 세부 항목으로는 힙 메모리 할당이 발생한 모든 파일과 라인 별[파일:라인] 메모리 사용량과 점유율이 사용량 기준으로 정렬되어 출력된다. 만약 --disable-force 옵션을 이용하여 메모리 모니터링 기능을 강제로 중지시킨 경우, 어떤 정보도 출력되지 않는다.
```

Reply:
When I see the https://github.com/CUBRID/cubrid/pull/5233, it looks print out 'Memory monitoring on the server has been disabled. You can use it again after restarting the server' message. And I think it is correct result.

Reply:
You're right. I change it.